### ISSUE TITLE
ALVR Handshake Error

### ISSUE THREAD

**Issue by KMThegamer2022** [alvr-org/ALVR#2566]
## Description

I can only connect to alvr sometimes because of a hand shake error that occurs 99% of the time 

## General Troubleshooting
- [ ✅] I carefully followed the instructions in the [README](https://github.com/alvr-org/ALVR/blob/master/README.md) and successfully completed the setup wizard
- [ ✅] I read the [ALVR GitHub Wiki](https://github.com/alvr-org/ALVR/wiki)

## Environment
Windows 11

### Hardware
**Note**: for Linux, an upload to the [`hw-probe`](https://linux-hardware.org/) database is preferred: `hw-probe -all -upload`

**CPU**:
AMD A8 APU With Radeon R5 Graphics
**GPU**:
AMD Radeon R5 Graphics
**GPU Driver Version**:
Not sure
**Audio**:
VB-Cable
### Installation
**ALVR Version**:
V20.11.1
**ALVR Settings File**:

**SteamVR Version**:

**Install Type**:
- [ ] Packaged (`exe`, `deb`, `rpm`, etc)
- [✅ ] Portable (`zip`)
- [ ] Source

**OS Name and Version** (`winver` on Windows or `grep PRETTY_NAME /etc/os-release` on most Linux distributions):

**Comment by The-personified-devil** [comment]
Can you provide a copy of the logs? Either enable the log to file option in the extra tab or just copy them from the log tab

**Comment by KMThegamer2022** [comment]
> Can you provide a copy of the logs? Either enable the log to file option in the extra tab or just copy them from the log tab



here is the log 


 [SESSION] Updated
15:50:14.010 [INFO] Server connected
15:50:14.425 [INFO] Initial socket buffer size: send: 65536B, recv: 65536B
15:50:15.046 [ERROR] Handshake error for 7569.client: An established connection was aborted by the software in your host machine. (os error 10053)
15:50:16.448 [INFO] Initial socket buffer size: send: 65536B, recv: 65536B
15:50:17.095 [ERROR] Handshake error for 7569.client: An established connection was aborted by the software in your host machine. (os error 10053)
15:50:18.525 [INFO] Initial socket buffer size: send: 65536B, recv: 65536B
15:50:19.144 [ERROR] Handshake error for 7569.client: An established connection was aborted by the software in your host machine. (os error 10053)
15:50:20.573 [INFO] Initial socket buffer size: send: 65536B, recv: 65536B
15:50:22.162 [ERROR] Handshake error for 7569.client: An established connection was aborted by the software in your host machine. (os error 10053)

**Comment by Aceleon123** [comment]
It is one of the most common error that I found, my way of fixing it (Sometimes works) is setting pc audio and quest mic to the same source, ALVR will freak out and sometimes it fixes connection.

Good luck!

**Comment by kayky21** [comment]
at first i fixed it by just disabling the "Headset microphone" option, but i wanted mic so then after fiddling around a little more, i found out you have to set CABLE OUTPUT as the default communication device and default device for it to work properly

**TLDR;
set CABLE OUTPUT as "Default Communication Device" + "Default Device" under "Recording", that seems to fix it**

**Comment by kayky21** [comment]
so im assuming by the two thumbs up that my suggestion worked?

**Comment by BiGB0YTED** [comment]
no it didnt @kayky21

**Comment by BiGB0YTED** [comment]
WDYM!!!😭😭

**Comment by kayky21** [comment]
> WDYM!!!😭😭

![Image](https://github.com/user-attachments/assets/fb62adca-9d53-4a04-adea-e7f83dcaf64f)

**Comment by Jurangren** [comment]
Try my fixed fork: [https://github.com/Jurangren/ALVR](https://github.com/Jurangren/ALVR)

There is a Windows release available in the fork.

Please let me know if it works properly for you.
